doc/api_assets/style.css

```
@@ -109,7 +156,7 @@ em code {
 
 #gtoc > ul > li {
   display: inline;
-  border-right: 1px #000 solid;
+  border-right: 1px var(--color-text-primary) solid;
```

I'd much prefer `currentColor` which is more expressive (we want the border to be same coolor as normal text) and less verbose. Is there any reason to use a variable here?
```suggestion
  border-right: 1px currentColor solid;
```